transformer/resources.py

```diff
@@ -47,7 +47,7 @@ class Subnote(odin.Resource):
 
 class Note(odin.Resource):
     type = odin.StringField(choices=resource_configs.NOTE_TYPE_CHOICES)
-    title = odin.StringField()
+    title = odin.StringField(null=True)
```

Comment:
Hmm I'm not sure I agree with this - this means that we'll have to do some work either in the indexer or the frontend to infer titles from type. I'd honestly do that work in the transformer.